lib/tasks/csv_load:all.rake

```diff
@@ -4,16 +4,16 @@ namespace :csv_load do
   desc "Run all CSV Imports"
   task :all => :environment do
     
-    ActiveRecord::Base.connection.tables.each do |t|
-      ActiveRecord::Base.connection.reset_pk_sequence!(t)
-    end
-
+    
     Rake::Task["csv_load:customers"].invoke
     Rake::Task["csv_load:merchants"].invoke
     Rake::Task["csv_load:invoices"].invoke
     Rake::Task["csv_load:items"].invoke
     Rake::Task["csv_load:invoice_items"].invoke
     Rake::Task["csv_load:transactions"].invoke
     
+    ActiveRecord::Base.connection.tables.each do |t|
```

Comment:
Glad this is working now. This should fix our deployment issues, and I think that this command makes all of the resetting in the individual rake files redundant.

But, you know what they say... two is one, and one is none